src/System.IO.Pipelines/src/System.IO.Pipelines.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configurations>netcoreapp-Debug;netcoreapp-Release;netcoreapp3.0-Debug;netcoreapp3.0-Release;netstandard-Debug;netstandard-Release</Configurations>
-    <DefineConstants Condition="'$(TargetGroup)'=='netstandard'">$(DefineConstants);INTERNAL_NULLABLE_ATTRIBUTES</DefineConstants>
+    <Configurations>netcoreapp-Debug;netcoreapp-Release;netcoreapp3.0-Debug;netcoreapp3.0-Release;netstandard2.0-Debug;netstandard2.0-Release</Configurations>
+    <DefineConstants Condition="'$(TargetsNetStandard)' == 'true'">$(DefineConstants);INTERNAL_NULLABLE_ATTRIBUTES</DefineConstants>
```

Comment:
Nit: I would rather use `TargetFramework` == `netstandard2.0` here as the `INTERNAL_NULLABLE_ATTRIBUTES` should only be included for `netstandard2.0` so if we add a ns2.1 to this project configuration would be problematic with the ones available in `netstandard.dll`.